frontend/src/modules/Environments/components/EnvironmentRedshiftConnectionAddForm.js

```diff
@@ -104,9 +115,25 @@ export const EnvironmentRedshiftConnectionAddForm = (props) => {
           Add a Redshift connection to environment {environment.label}
         </Typography>
         <Typography align="center" color="textSecondary" variant="subtitle2">
-          The Redshift connection is owned by the selected Team. It is used to
-          import Redshift Datasets.
+          The Redshift connection contains the metadata to connect to Redshift
+          with a particular user.
         </Typography>
+      </Box>
+      <Box sx={{ p: 3 }}>
+        <Box sx={{ pl: 5, pr: 5 }}>
+          <Alert
+            severity="info"
+            variant="outlined"
+            sx={{ whiteSpace: 'pre-line' }}
+          >
+            <AlertTitle>Types of Connections</AlertTitle>- DATA USER: This
```

Comment:
I am not a big fun of this chunky text appearing at the top, I'd enhance and keep only the tooltips per option and maybe even maybe make the longer text appear if you click a button.

Something likes [this](https://stackoverflow.com/questions/77184297/how-to-embed-a-tooltip-icon-in-a-select-material-ui) or [this](https://stackoverflow.com/questions/59154960/how-to-add-icons-and-tooltip-on-react-select-component)

Reply:
Good suggestions! I made some changes. I tried to implement the tooltip for the complete select row, but it was struggling to update the value once selected. That's why I changed it to the word only. But i think with the info icon we are good